src/Microsoft.DocAsCode.MarkdigEngine/MarkdownExtensions.cs

```diff
@@ -213,5 +213,17 @@ public static MarkdownPipelineBuilder UseYamlHeader(this MarkdownPipelineBuilder
             pipeline.Extensions.Insert(0, new YamlHeaderExtension());
             return pipeline;
         }
+
+        public static MarkdownPipelineBuilder UseRow(this MarkdownPipelineBuilder pipeline)
+        {
+            pipeline.Extensions.Insert(0, new RowExtension());
```

Comment:
>Insert [](start = 32, length = 6)

No need to insert it as the first extension, use `AddIfNotAlready` instead. The same as `UseNestedColumn`.